src/components/EnvironmentBadge.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {Text, View} from 'react-native';
+import styles from '../styles/styles';
+import CONFIG from '../CONFIG';
+import CONST from '../CONST';
+
+const EnvironmentBadge = () => {
+    // If we are on production, don't show any badge
+    if (CONFIG.EXPENSIFY.ENVIRONMENT === CONST.ENVIRONMENT.PRODUCTION) {
+        return;
+    }
+
+    const badgeText = CONFIG.EXPENSIFY.ENVIRONMENT === CONST.ENVIRONMENT.STAGING ? 'STG' : 'DEV';
+    return (
+        <View
+            style={[styles.badge, styles.badgeSuccess, styles.ml2]}
```

Comment:
Nice, I think we just need to add a style for `badgeDanger` so that we can get the red DEV badge.